client/src/components/CommanderTeamView.js

```diff
@@ -4,6 +4,8 @@ import Footer from './Footer';
 import PenstateUniversity from './PenstateUniversity';
```

Comment:
@eralpsahin @devGyan  What is this variable for?

Reply:
It is the reusable component.
![Screenshot from 2019-09-19 10-06-53](https://user-images.githubusercontent.com/53865713/65213867-77830b00-dac5-11e9-86a1-a329eb61ee55.png)

Reply:
it is used in team admin as well as organization admin.